enterprise/frontend/src/metabase-enterprise/collections/constants.ts

```diff
@@ -14,7 +14,7 @@ export const REGULAR_COLLECTION: CollectionAuthorityLevelConfig = {
 export const OFFICIAL_COLLECTION: CollectionAuthorityLevelConfig = {
   type: "official",
   name: t`Official`,
-  icon: "badge",
+  icon: "verified_collection",
```

Comment:
Let's use Vitorio's new SVG for verified collections

Reply:
let's get a design review on this. I like it, but it has impact beyond this area of the product.

![Screen Shot 2024-09-26 at 2 32 49 PM](https://github.com/user-attachments/assets/04395d5b-06ec-4bc1-836b-92679faffcf3)

![Screen Shot 2024-09-26 at 2 33 41 PM](https://github.com/user-attachments/assets/66058e0c-5663-42a4-99ba-9b844248e909)

Reply:
`find frontend enterprise e2e -type f | xargs sed -i "s/verified_collection/official_collection/g"`
easy peasy